lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -264,7 +264,7 @@ def define
       outcome :outcome_opposite_sex_marriage_other_countries
 
       #CP outcomes
-      outcome :outcome_cp_or_equivalent
+      outcome :outcome_same_sex_civil_partnership_or_equivalent
```

Comment:
Minor: Although technically correct, I wonder how useful the `_or_equivalent` bit of the name is.